### ISSUE TITLE
"Learn More" links are WCAG 2.1 SC 2.5.3 violations

### ISSUE THREAD

**Issue by aardrian** [w3c/aria-practices#2338]
There are 8 "Learn More" links on the home page at https://www.w3.org/WAI/ARIA/apg/:
`//a[contains(text(),"Learn More")]`

Each has an `aria-label` that differs from the visible text:
`//a[contains(text(),"Learn More")][@aria-label]`

These violate SC [2.5.3: Label in Name](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html).

Example:
```
<a href="practices/landmark-regions/" aria-label="View landmarks guide" class="button-link">
  Learn More
</a>
```

The accName is "View landmarks guide" but the visible link text is "Learn More".

Identified by Meagan Eller in the a11y Slack channel.

Editing to note that those opening code blocks are XPath searches you can do to find all the nodes in the DOM view of your browser dev tools. More info if anyone reading this is new to it: _[XPath for In-Browser Testing](https://adrianroselli.com/2021/04/xpath-for-in-browser-testing.html)_

**Comment by shirsha** [comment]
@aardrian 
An issue is raised for this :)
https://github.com/w3c/aria-practices/issues/2328

**Comment by aardrian** [comment]
Oops. I filtered for issues _since_ launch. It did not occur to me to look for an issue for a 2.5.3 violation filed _prior_ to launch. I guess I figured a WAI site would not launch with known WCAG violations.

**Comment by aardrian** [comment]
Sorry, meant to close as duplicate, not completed.
